src/analysis/credData.js

```diff
@@ -74,7 +71,7 @@ export type EdgeCredOverTime = {|
 export function computeCredData(
   scores: TimelineCredScores,
   nodeOrder: $ReadOnlyArray<NodeAddressT>,
-  dependencyPolicies: $ReadOnlyArray<DependencyMintPolicy>
+  dependencyPolicies: $ReadOnlyArray<BonusPolicy>
```

Comment:
Should rename the param to `bonusPolicies` as well

Reply:
Done